src/services/posts/unflag-post.test.js

```diff
@@ -0,0 +1,41 @@
+const {expect} = require('chai')
+const db = require("../../db")
+const flagPost = require("./flag-post")
+const unflagPost = require("./unflag-post")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('UNFLAG a post', () => {
+	let user
+	
+	before(async ()=>{
+		user = await fixtures.insertUser() 
+	})
+	describe('GIF', () => {
+		let post
+		before(async ()=>{
+			post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'gif'
+			})
+		})
+		it('should delete a gif flag in the database', async () => {
+			await fixtures.insertPost({
+			   userId : user.id , 
+			   type : 'gif'
+		   })
+			   await flagPost({ 
```

Comment:
@itzgeebee lets insert the flag post with a utils function instead

 we don't want to couple another function while testing another function hence the utils functions to set up the scenarios

 otherwise if that service function doesn't work and isn't flagging anymore, this will still pass